service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorController.java

```diff
@@ -15,24 +15,42 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.ge.predix.acs.commons.web.AcsApiUriTemplates;
 import com.ge.predix.acs.commons.web.BaseRestApi;
 import com.ge.predix.acs.commons.web.RestApiException;
 import com.ge.predix.acs.rest.AttributeConnector;
 
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;;
+
+
 @RestController
 public class AttributeConnectorController extends BaseRestApi {
 
     @Autowired
     private AttributeConnectorServiceImpl service;
 
+    @ApiOperation(
+            value = "Creates or updates connector configururation for external resource attributes for the given zone.",
+            tags = { "Attribute Connector Management" },
+            response = AttributeConnector.class)
+    @ApiResponses(
+            value = { @ApiResponse(
+                    code = 201,
+                    message = "Connector configuration for the given zone is successfully created.") })
     @RequestMapping(
             method = PUT,
             value = V1 + AcsApiUriTemplates.RESOURCE_CONNECTOR_URL,
-            consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<AttributeConnector> putResourceConnector(@RequestBody final AttributeConnector connector) {
+            consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I thought we return an empty body for upserts in which case this isn't needed, no?